### PR TITLE
Eliminate wrapperProps

### PR DESCRIPTION
**Pull request by jackkav**
this is a pre-fc pass to cut some  700 lines, from app and props of yesteryear, @gatzjames is working on a FC pass on app.tsx so hoping this will help. Related #4937

~~next step would be to rethink the pagelayout component and try to nest the mouse handlers and pan height and width story within rather than drilling from app.tsx through wrapperProps. #4994~~

highlights
- moved each function from app.tsx and wrapper.tsx to "appropriate" place in component tree.
- I exercised some discretion about where is appropriate based on props and selectors available in a given context.
- avoided any which involved react rendering hacks.
- curiously send and download request behaviour  is coupled to redux state.

### CODE REVIEW

File: packages/insomnia/src/ui/components/editors/request-parameters-editor.tsx [hunk 1]
```diff
@@ -1,22 +1,21 @@
...
 }
 
 export const RequestParametersEditor: FC<Props> = ({
-  onChange,
   request,
   bulk,
 }) => {
   const handleBulkUpdate = useCallback((paramsString: string) => {
-    const params: {
+    const parameters: {
```
Comment (gatzjames) [inline packages/insomnia/src/ui/components/editors/request-parameters-editor.tsx:18]: ⚡️

File: packages/insomnia/src/ui/components/viewers/response-viewer.tsx [hunk 2]
```diff
@@ -19,7 +19,6 @@ import { executeHotKey } from '../../../common/hotkeys-listener';
 import { xmlDecode } from '../../../common/misc';
 import { CodeEditor, UnconnectedCodeEditor } from '../codemirror/code-editor';
 import { KeydownBinder } from '../keydown-binder';
```
Comment (gatzjames) [inline packages/insomnia/src/ui/components/viewers/response-viewer.tsx:21]: 😌

File: packages/insomnia/src/ui/components/page-layout.tsx [hunk 3]
```diff
@@ -23,7 +21,6 @@ const Pane = forwardRef<HTMLElement, { position: string; children: ReactNode }>(
 );
 
 interface Props {
-  wrapperProps: WrapperProps;
```
Comment (gatzjames) [inline packages/insomnia/src/ui/components/page-layout.tsx:26]: So nice to see this go

**Review by filfreire — COMMENTED** [review]
Heads up @jackkav, found a number of issues ⚠️ 

1) When we first open the app, I'm not able to send a request in any way (via keyboard or clicking send button)

2) When I git sync an existing repository, suddenly I'm now able to send a request anywhere, BUT, the `Branches` modal doesn't open at all. In the console we get an error like: 
```
Uncaught Error: Modal was not registered with the app
    at VSe (index.ts:53:11)
    at xn (index.ts:23:10)
    at dq._handleManageBranches (git-sync-dropdown.tsx:240:5)
    at H4._handleClick (dropdown-item.tsx:36:7)
```
branches



3) The `Export Data` modal doesn't open and returns an error:
```
Uncaught Error: Modal was not registered with the app
    at VSe (index.ts:53:11)
    at xn (index.ts:23:10)
    at import-export.tsx:50:5
    at H4._handleClick (dropdown-item.tsx:36:7)
    at Object.tue (react-dom.production.min.js:52:317)
```
export

**Comment by jackkav** [comment]
@filfreire fixed! Thanks! 🙏

**Review by filfreire — APPROVED** [review]
image

**Pull request merged**